### ISSUE TITLE
benchmark release note

### ISSUE THREAD

**Issue by dzhwinter** [PaddlePaddle/Paddle#8533]
Release v0.11.0 includes a new feature PaddlePaddle Fluid, Fluid is designed to allow users to program like PyTorch and TensorFlow Eager Execution. https://github.com/PaddlePaddle/Paddle/releases/tag/v0.11.0

To validate our Fluid performance, we implement 5 benchmark models and compare the converge speed and training speed with tensorflow. https://github.com/dzhwinter/benchmark

1. Model convergence rate comparison. 
We provide the training curves in VisualDL graph https://github.com/PaddlePaddle/Paddle/VisualDL
Then we can check every minibatch accuracy/loss during the training process.
2. Model speed comparison. 
We compare the training speed in different batch size, to measure the framework performance in different situations.

Here I mark some fixes should be done before release.
* common
- [x] loss, accuracy graph should be put on the same page, for highlight the difference.
- [x] TensorFlow GPU memory occupation also sum up of the GPU driver, but Fluid only show the real memory. Need to be fixed.
- [ ] paddlepaddle.org will add a link for showing the result.
- [x] mnist GPU. Fluid should skip the first several batches, for measuring the result.
- [ ] lstm CPU batch=32, paddle fluid curve is not stable，it happens randomly.
- [x] seq2seq fluid log should be updated.
- [x] resnet GPU is not same with CPU identically
- [x] delete vgg16 flowers cpu data, it does not show the correct result.
- [ ] vgg gpu fluid speed difference is not proportional to other models in TitanX/P40c.

* vgg16
- [x] cifar10 GPU batchsize=64, 128，speed result should be checked.
- [x] vgg16 check the difference of batch norm op or fused batch norm op.
- [x] vgg16 batchsize=64 speed should be checked.

* resnet50
- [x] GPU batch 64 run well  at docker image release/0.11.0，but the latest code may OOM。@董志宏

* stacked lstm
- [x] may occur the overfit problem in Fluid lstm scripts.

* seq2seq
- [x] GPU batch=64, 128 GPU may OOM at the latest docker image.

* visusalDL
- [x] click may not refresh the page.
- [x]  seq2seq zoom in error.
- [x]  the latest version should be installed inside docker.

**Comment by dzhwinter** [comment]
2018-3-1 
TODO(dzhwinter):
- [ ] sequence_pool op enhance speed
- [ ] fix the parallel.do bug/ memory leak bug 
- [ ]  release benchmark result.

**Comment by peterzhang2029** [comment]
2018-3-1 
TODO(zhangchao)
- [ ] Add rnn search with multi-thread and test the speedup.
- [ ] Complete the profiling of Transformer.

**Comment by dzhwinter** [comment]
The benchmark job has been done.
